custom/openclinica/utils.py

```diff
@@ -266,15 +266,13 @@ def get_tz_mins(tz_str):
 
 def oc_format_time(time_str, to_timezone, date_=None):
     """
-    Format time strings for OpenClinica
-
-    .. NOTE:: This uses 1900-01-01 as `date_` if no date is given. This can have implications for daylight savings.
```

Comment:
This is not true, because daylight savings timezones have different names (e.g. CET vs CEST; EST vs EDT; etc.).